tests/conftest.py

```diff
@@ -110,7 +110,7 @@ async def kakarot(
     return await starknet.deploy(
         source="./src/kakarot/kakarot.cairo",
         cairo_path=["src"],
-        disable_hint_validation=False,
+        disable_hint_validation=True,
```

Comment:
Why do you remove the hint_validation?

Reply:
because for testing when you want to `print` or `breakpoint` you need to write hints, and each time one needs to disable then enable again. So for testing I think that it's easier not to validate the hint at all. In any case we don't write hint and this wouldn't pass the compile CI step